pkg/apis/kops/networking.go

```diff
@@ -257,6 +257,9 @@ type CiliumNetworkingSpec struct {
 	EnableTracing bool `json:"enableTracing,omitempty"`
 	// EnablePrometheusMetrics enables the Cilium "/metrics" endpoint for both the agent and the operator.
 	EnablePrometheusMetrics bool `json:"enablePrometheusMetrics,omitempty"`
+	// EncryptionEnabled enables Cilium Encryption.
+	// Default: false
+	EncryptionEnabled bool `json:"encryptionEnabled"`
```

Comment:
To be more inline with the keys above, I suggest renaming to `EnableEncryption`

Reply:
Fixed in https://github.com/kubernetes/kops/pull/9154/commits/41c76dabaad8ffcbef6a4b19dfdf1174325bd8f8